diff --git a/node_modules/@shopify/react-native-skia/android/build.gradle b/node_modules/@shopify/react-native-skia/android/build.gradle
index c5431ee..28c0622 100644
--- a/node_modules/@shopify/react-native-skia/android/build.gradle
+++ b/node_modules/@shopify/react-native-skia/android/build.gradle
@@ -235,6 +235,10 @@ android {
         extractHeaders
         extractJNI
     }
+
+    buildFeatures {
+        buildConfig = true
+    }
 }
 
 repositories {
