diff --git a/node_modules/react-native-vector-icons/android/build.gradle b/node_modules/react-native-vector-icons/android/build.gradle
index 3e615e9..834271a 100644
--- a/node_modules/react-native-vector-icons/android/build.gradle
+++ b/node_modules/react-native-vector-icons/android/build.gradle
@@ -38,6 +38,9 @@ android {
             }
         }
     }
+    buildFeatures {
+        buildConfig = true
+    }
 }
 
 repositories {
